manifests/init.pp

```diff
@@ -115,14 +122,24 @@
 
   if $nfs_service == 'USE_DEFAULTS' {
     $nfs_service_real = $default_nfs_service
+    $nfs_service_required_svcs_real = $default_nfs_service_required_svcs
   } else {
     $nfs_service_real = $nfs_service
+    $nfs_service_required_svcs_real = undef
   }
 
   package { $nfs_package_real:
     ensure => present,
   }
 
+  if $nfs_service_required_svcs_real and $nfs_service_real {
+   service { $nfs_service_required_svcs_real:
```

Comment:
In Puppet resource declarations, supplying an array in the title attribute has same effect as declaring a separate resource for each item in the array.  I.e. it would invoke svcadm multiple times.  Supplying an array to the "name" attribute apparently only casts the array into a concatenated string that is then supplied to the underlying package provider tool (e.g. svcadm).